stackslib/src/chainstate/nakamoto/shadow.rs

```diff
@@ -728,6 +729,7 @@ impl NakamotoBlockBuilder {
             1,
             None,
             None,
+            u64::from(DEFAULT_MAX_TENURE_BYTES),
```

Comment:
Let's instead make this 0.  Miners cannot produce shadow tenures, and nodes will not accept shadow tenures from the network (they can only be instantiated from hardwired code).

Reply:
Is there a reason why this isn't zero?

Reply:
Because the code does not give special meaning to 0: 
```
if total_tenure_size >= self.max_tenure_bytes {
        return TransactionResult::skipped_due_to_error(tx, Error::TenureTooBigError);
}
```

so setting it to zero makes every attempt to fail